src/mount.initrd_zfs.c

```diff
@@ -205,6 +205,7 @@ int main(int argc, char *argv[]) {
 		exit(1);
 	}
 
+	ret = -1;
```

Comment:
Can you revert these changes? I'd like to properly fix it by returning what ZFS returned to me

Reply:
Sure.